packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,97 @@ void main() {
+          final List<Widget> widgetList = <Widget>[
+            Container(key: const Key('1'), child:  const Icon(Icons.account_circle))
+          ];
+          final SingleChildScrollView scrollView = SingleChildScrollView(
+            child: SizedBox(
+              height: 100,
+              width: 100,
+              child: ReorderableListView(
+                key: const Key('list'),
+                physics: const AlwaysScrollableScrollPhysics(),
+                onReorder: (int oldIndex, int newIndex) {},
+                children: widgetList,
+                scrollController: ScrollController(),
+              )
+            ),
+          );
+          await tester.pumpWidget(MaterialApp(
+          home: SizedBox(
+            height: 100,
+            width: 100,
+            child: scrollView,
+          ),
+        ));
+
+          final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await gesture.moveBy(_kGestureOffset);
+          // Move back to original position.
+          await gesture.moveBy(Offset(-_kGestureOffset.dx, -_kGestureOffset.dy));
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          final TestGesture overscrollGesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await overscrollGesture.moveBy(_kGestureOffset);
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          expect(tester.takeException(), isNull);
+        });
+
+        testWidgets('Default ReorderableListViews are not always scrollable', (WidgetTester tester) async {
+          final ReorderableListView view = ReorderableListView(
+            scrollDirection: Axis.horizontal,
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+          expect(view.physics, isNot(isInstanceOf<AlwaysScrollableScrollPhysics>()));
+        });
+
+        testWidgets('physics:AlwaysScrollableScrollPhysics overrides default behavior', (WidgetTester tester) async {
+          bool scrolled = false;
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Directionality(
+              textDirection: TextDirection.ltr,
+              child: NotificationListener<OverscrollNotification>(
+                onNotification: (OverscrollNotification message) { scrolled = true; return false; },
+                child: ReorderableListView(
+                  physics: const AlwaysScrollableScrollPhysics(),
+                  children: const <Widget>[],
+                  onReorder: (int oldIndex, int newIndex) {},
+                ),
+              ),
+            ),
+            )
+          );
+          await tester.dragFrom(const Offset(100.0, 100.0), const Offset(0.0, 100.0));
+          expect(scrolled, isTrue);
```

Comment:
Check to see if this test still passes even if you omit passing in `AlwaysScrollableScrollPhysics`. I have a feeling that by default, the `ReorderableListView` will use the platform defaults instead of defaulting to not having any over-scroll physics at all.

If so, you will probably have to rewrite this test to check that the particular scroll physics that is being passed in overrides whichever platform default would be used if nothing were passed into `ReorderableListView.physics`.